pyproject.toml

```diff
@@ -49,8 +49,8 @@ issues = "https://github.com/instructlab/instructlab/issues"
 "show" = "instructlab.config.show:show"
 
 [project.entry-points."instructlab.command.data"]
-"generate" = "instructlab.data.generate:generate"
-"list" = "instructlab.data.list:list_data"
+"generate" = "instructlab.cli.data.generate:generate"
```

Comment:
does the alias section also need to be altered? line 73

Reply:
good catch! updated